common/build.gradle

```diff
@@ -169,6 +169,10 @@ dependencies {
     testImplementation 'org.json:json:20160810'
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
     testImplementation "org.mockito:mockito-core:$rootProject.ext.mockitoCoreVersion"
+
+    testImplementation("org.mockito.kotlin:mockito-kotlin:4.1.0") {
+        exclude group: 'org.jetbrains.kotlin', module: 'kotlin-stdlib'
```

Comment:
Can you add comments about why this exclude is needed?

Reply:
Removed this exclude